src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main class="main">
+      <div class="boxs">
+        <form
+          action="#"
+          data-qa="big"
+          method="POST"
+          class="box"
+        >
+          <div href="#" class="box__link"></div>
```

Comment:
The 'div' element does not support the 'href' attribute. Consider using an 'a' tag instead if you want to create a link.